cypress/e2e/gui/gui-engage-sphere.spec.cy.js

```diff
@@ -0,0 +1,190 @@
+const API_URL = Cypress.env('API_URL');
+const CUSTOMERS_API_URL = `${API_URL}/customers`;
```

Comment:
Ao deixar uma linha em branco entre o fim da deinição das variáveis e o início da suite de testes, fica mais fácil identificar onde "uma coisa acaba" e "outra começa".

```suggestion
const CUSTOMERS_API_URL = `${API_URL}/customers`;

```